src/test/java/org/folio/rest/api/ItemShelvingOrderMigrationServiceApiTest.java

```diff
@@ -53,7 +49,7 @@ public void shouldStopProcessingWhenCannotConvertItem() throws Exception {
   private JsonObject getTenantAttributes() {
     return JsonObject.mapFrom(new TenantAttributes()
       .withModuleFrom("20.1.1")
-      .withModuleTo("20.2.0")
+      .withModuleTo("20.2." + nextPatch.incrementAndGet())
```

Comment:
Why does this process start with version `20.2.1` rather than `20.2.0` (due to the AtomicInteger starting at 1)?

Why does this need to be incremented each time?

Reply:
Because module already on 20.2.0.

We have to increment them because the method is used by multiple tests but the tenant is for a entire test class.

When 1st test passed tenant is on 20.2.1, then second should be on 20.2.2, etc.

Reply:
Oh, these tests are pretending to enable versions 20.2.1, 20.2.2 etc to mimic enabling 20.2.0?

Reply:
yes, correct.

Reply:
Could they at least start from `20.2.0` so they also issue the expected instructions?

Reply:
Unfortunately no, because module already on 20.2.0 after tenant set-up by @BeforeAll method.

Reply:
However there is a unit test `org.folio.services.migration.item.ItemShelvingOrderMigrationServiceTest` that demonstrates that migration is triggered for 20.1.0 -> 20.2.0 migration.